sanic/config.py

```diff
@@ -201,7 +201,11 @@ def load_environment_vars(self):
         for k, v in os.environ.items():
             if k.startswith(SANIC_PREFIX):
                 _, config_key = k.split(SANIC_PREFIX, 1)
-                if v.isdigit():
-                    self[config_key] = int(v)
+                # This is a float or an int
```

Comment:
nvm, you didn't catch the error, just `pass`, so there are not much differences, then.

It's much more expensive to `catch` the exception.